src/main/scala/ai/privado/audit/DataElementDiscovery.scala

```diff
@@ -126,11 +128,41 @@ object DataElementDiscovery {
     collectionInputList.toList
   }
 
+  def getCollectionMethodInfo(xtocpg: Try[Cpg]): Map[String, ListBuffer[CollectionMethodInfo]] = {
+    // className -> (MethodName, Endpoint)
+    val collectionMethodInfoMap = new mutable.HashMap[String, ListBuffer[CollectionMethodInfo]]()
+    xtocpg match {
+      case Success(cpg) => {
+        val parameterList = cpg.parameter.where(_.tag).l
```

Comment:
For collectionPoints parameter, update the query to 
```
cpg.parameter.where(_.method.tag.nameExact(CatLevelOne.Collections.name)).l
```

Update this query every where